# Request a specific guest to follow the robot
string guest_name
string robot_name
---
bool success
